let User,CustomErrorHandler;_4df‍.x([["default",()=>_4df‍.o]]);_4df‍.w("../../models",[["User",["User"],function(v){User=v}]]);_4df‍.w("../../services/customErrorHandler",[["default",["CustomErrorHandler"],function(v){CustomErrorHandler=v}]]);


const userController = {
    async me(req, res, next) {
        try {
            const user = await User.findOne({ __id: req.user.__id }).select('-password -createdAt -updatedAt -__v');
            if (!user) {
                return next(CustomErrorHandler.notFound());
            }
            res.json(user);
        } catch (err) {
            return next(err);
        }
    }
}

_4df‍.d(userController);